policyengine_us/parameters/gov/states/vt/tax/income/rates/joint.yaml

```diff
@@ -4,6 +4,7 @@ metadata:
   threshold_unit: currency-USD
   rate_unit: /1
   label: Vermont State income tax rates for joint filers
+  uprating: gov.irs.uprating
```

Comment:
we have to add the uprating here to each threshold individually (we dont want the rate to be updated each year)
- see the CA PR files:
https://github.com/PolicyEngine/policyengine-us/pull/3770/files#diff-ab9344c8a2727114a1c0a5e36747a11cf0ad22885047cc17c26aba08d552645a

Reply:
same for all marginal rate or single amount parameters

Reply:
Could you check whether the values are rounded or not?